client/indexer-service/src/handler.rs

```diff
@@ -212,16 +212,82 @@ impl IndexerService {
                 )
                 .await?;
             }
-            pallet_file_system::Event::BspConfirmStoppedStoring { .. } => {}
-            pallet_file_system::Event::BspConfirmedStoring { .. } => {}
+            pallet_file_system::Event::BspConfirmStoppedStoring {
+                bsp_id: _,
+                file_key,
+                new_root: _,
+            } => {
+                File::delete(conn, file_key.as_ref().to_vec()).await?;
```

Comment:
Isn't this deleting the file from the database if just one BSP stops storing it? That would be wrong.